devtools/projects/ng-devtools-backend/src/lib/client-event-subscribers.ts

```diff
@@ -211,10 +213,15 @@ const getNestedPropertiesCallback =
 // Subscribe Helpers
 //
 
-// todo: parse router tree with framework APIs after they are developed
 const getRoutes = (messageBus: MessageBus<Events>) => {
-  // Return empty router tree to disable tab.
-  messageBus.emit('updateRouterTree', [[]]);
+  const forest = prepareForestForSerialization(
+    initializeOrGetDirectiveForestHooks().getIndexedDirectiveForest(),
+    ngDebugDependencyInjectionApiIsSupported(),
+  );
+  const rootInjector = forest[0].resolutionPath?.find((i) => i.name === 'Root');
```

Comment:
If `ngDebugDependencyInjectionApiIsSupported()` is false then there will be no resolution paths in the forest nodes. We should handle this case by doing `forest[0].resolutionPath?.find?.(...)` or the suggestion below

```suggestion
  const rootInjector = (forest[0].resolutionPath ?? []).find((i) => i.name === 'Root');
```